conductr_cli/conduct_main.py

```diff
@@ -386,6 +389,20 @@ def build_parser(dcos_mode):
 
     agents_parser.set_defaults(func=conduct_agents.agents)
 
+    # Sub-parser for `load-license` sub-command
+    load_license_parser = subparsers.add_parser('load-license',
+                                                help='Obtains license from Lightbend.com and '
```

Comment:
I want it to be a single line actually as per the output below. I had the line break there to prevent warning due to line length exceeding a certain acceptable width:

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ conduct --help
usage: conduct [-h]
               {version,info,service-names,acls,load,run,stop,unload,events,logs,setup-dcos,deploy,members,agents,load-license}
               ...

optional arguments:
  -h, --help            show this help message and exit

commands:
  {version,info,service-names,acls,load,run,stop,unload,events,logs,setup-dcos,deploy,members,agents,load-license}
                        Use one of the following sub commands:
    version             Print version
    info                Print bundle information
    service-names       Print the service names available to the service locator
    acls                Print request ACL information
    load                Load a bundle
    run                 Run a bundle
    stop                Stop a bundle
    unload              Unload a bundle
    events              Show bundle events
    logs                Show bundle logs
    setup-dcos          Setup integration with the DC/OS CLI
                        so that 'dcos conduct ..' commands can
                        be used to access ConductR via DC/OS
    deploy              Replaces a running bundle with a deployed version
    members             Display cluster member information
    agents              Display agent information
    load-license        Obtains license from Lightbend.com and loads the license into ConductR
```